docs/en/02_Developer_Guides/00_Model/11_Scaffolding.md

```diff
@@ -154,23 +167,28 @@ class Player extends DataObject
 
 ```
 
-Use a single search field that matches on multiple database fields with `'match_any'`
+### Searching many db fields on a single search field
+
+Use a single search field that matches on multiple database fields with `'match_any'`. This also supports specifying a field and a filter, though it is not necessary to do so.
 
 ```php
 class Order extends DataObject
 {
+    private static $db = [
+        'Name' => 'Varchar',
+    ];
+
     private static $has_one = [
         'Customer' => Customer::class,
         'ShippingAddress' => Address::class,
     ];
 
     private static $searchable_fields = [
-        'CustomFirstName' => [
+        'Name' => [
```

Comment:
Note that trying to use some custom field name here will result in the following exception:
> Uncaught LogicException: CustomFirstName is not a relation/field on Order